themebuilder/modules/metadata-loader.js

```diff
@@ -0,0 +1,32 @@
+class MetadataLoader {
+    constructor() {
+        this.fullMetadata = require("../data/metadata/dx-theme-builder-metadata");
+    }
+
+    groupMetadata(metadata) {
+        let groups = {};
+        metadata.forEach(item => {
+            if(!groups[item.Group]) {
+                groups[item.Group] = [];
+            }
+
+            groups[item.Group].push(item);
+        });
+
+        return groups;
+    };
+
+    load(theme, colorScheme) {
+        let that = this;
+        return new Promise((resolve) => {
+            var metadata = this.fullMetadata[theme + "_" + colorScheme.replace("-", "_") + "_metadata"];
```

Comment:
Should it be `let`?

Reply:
Actually, there is no needed to use `that` variable. It's ES5 legacy. You already used an arrow function, that does not change the context.